docs/variants/asus_kgpe_d16/setup.md

```diff
@@ -0,0 +1,214 @@
+available with qspimux. Flashing may be done via the commands shown earlier or
+the flash script on present in the root directory of the RTE:
+
+```shell
+./flash.sh coreboot.rom
+```
+
+The script will automatically power off the board and update flash chip
+contents using flashrom. After the operation is finished the routing of SPI
+signals is restored to the platform.
+
+#### Power management
+
+RTE is able to control the power of the board with the `rte_ctrl` application
+or with the web GUI available at RTE's IP address port 8000. Possible command are:
+
+```shell
+./rte_ctrl -pon # power on the platform
+./rte_ctrl -poff # force 5seconds power button power off
+./rte_ctrl -reset # briefly assert platform reset pin to perform reset
+```
+
+It is also possible to control the ATX power supply by disconnecting it from
+the power socket. For this special purpose Sonoff S20 is used to remotely
+control the power socket activity. Sonoff IP to RTE IP mapping:
+
+* RTE IP 192.168.4.236 - Sonoff IP 192.168.4.125
+* RTE IP 192.168.4.111 - Sonoff IP 192.168.4.146
+
+Sonoffs respond to rest API requests to enable or disable the power socket.
+Shell scripts has been added to RTE root directories to ease the ocntrol of
+power of the KGPE-D16 platform:
+
+```shell
+#!/bin/bash
+
+SONOFF_IP=192.168.4.125
+
+if [[ "$1" == "off" ]]; then
+    wget -q -O - http://$SONOFF_IP/switch/sonoff_s20_relay/turn_off --method=POST
+elif [[ "$1" == "on" ]]; then
+    wget -q -O - http://$SONOFF_IP/switch/sonoff_s20_relay/turn_on --method=POST
+elif [[ "$1" == "show" ]]; then
+    wget -q -O - http://$SONOFF_IP/switch/sonoff_s20_relay
+    echo -e '\n'
+else
+    echo -e "\$1 == on|off|show|toggle\nEdit this script to set the sonoff ip."
+    echo -e 'Current state:'
+    wget -q -O - http://$SONOFF_IP/switch/sonoff_s20_relay
+    echo -e '\n'
+fi
+```
+
+To turn off the pwoer from the platform `./sonoff.sh off` and to turn it on
+`./sonoff.sh on`. Invoking the script without any argument will print the
+current state of the switch.
+
+#### Serial console access
+
+Serial port is very useful in debugging and as a remote console of the system
+on the platform. The serial port of KGPE-D16 is connected to the RTE's serial
+port which can be accessed on RTE with `/dev/ttyS1`. One may use minicom
+directly or take advantage of the ser2net service running on RTE which exposes
+the serial port over the network. The serial port can be accessed remotely like
+this `telnet RTE_IP 13541`, e.g.:
+
+```shell
+telnet 192.168.4.236 13541
+```
+
+### Demo
+
+A short demo presenting RTE capabilities with KGPE-D16:
+
+<script id="asciicast-438342" src="https://asciinema.org/a/438342.js" async data-rows="25"></script>
+
+### Remote access
+
+if you are interested in developing coreboot on KGPE-D16 and have some spare
+time, there is a possibility to obtain the access to the hardware. Drop us an
+email to [leads@3mdeb.com](mailto:leads@3mdeb.com) with a request.
```

Comment:
@miczyg1 in long run we looking to extend this section: https://docs.dasharo.com/inquiries/overview/